mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = async (req, res) => {
+  try {
+    const deletionResult = await Order.findOneAndDelete({ _id: req.params.orderId });
+    if (!deletionResult) {
+      return res.status(404).json({ error: "Order not found" });
+    }
+    res.json({ message: "Order removed successfully" });
+  } catch (error) {
```

Comment:
The `catch` block should be consistent with the rest of the error handling in the code. In other functions, error handling is done using `if (err) { ... }` instead of `catch (error) { ... }`. For consistency, it's better to use the same error handling approach throughout the code.